src/features/list-view/styles/ListView.scss

```diff
@@ -20,16 +22,24 @@
         text-transform: none;
         width: 100%;
 
-        .bdl-icon-sort-chevron.bdl-ListView-isSortAsc {
+        &--sortable {
+            cursor: pointer;
+        }
+
+        .bdl-icon-sort-chevron.#{$prefix}-isSortAsc {
             transform: rotate(180deg);
         }
     }
 
-    .bdl-ListView-columnCell {
+    .#{$prefix}-columnCell {
```

Comment:
This isn't really necessary, in my opinion. Repeating the class name is acceptable. It could also probably be shortened to `.bdl-ListView`, since that's the name of the component.

Reply:
Final CSS will also have full classname, so having it a variable makes no difference.

Reply:
"ListView" is too generic. There is already a different ListView in EUA which could be moved to BUI in the near future.

Reply:
Then the component files, folder, etc. should also be renamed to something more specific. Also, SUIT uses hyphens as indicators of descendent components.

Reply:
I guess I have two options:
1. Leave as bdl-ListView and risk a future collision
2. Rename the prefix but introduce a breaking change

Given my time constraints, I'll have to go with #1.

Reply:
@priyajeet It's an eyesore for readers to repeat a lengthy prefix but if we prefer searchability, then I can buy into that.